public/manifest.json

```diff
@@ -0,0 +1,5 @@
+{
+  "id": "com.github.jankaritech.web.mdpresentation",
```

Comment:
`id` line should be removed

Reply:
I was using it in other files but yeah better move it to index.ts. Thanks

Reply:
Ah, No. we need it in vite config as well. Could be a trouble to put it in src/index.ts

@kulmann Does ocis process `id` in manifest file?

Reply:
oCIS doesn't process an `id` prop from the manifest.

Reply:
:+1: Thanks